### DISCUSSION TITLE
impossible to push

### DISCUSSION THREAD

**Discussion by ErwanPrigent1** [community/community#28396] [Other Feature Feedback, Questions, & Ideas]
Dear all

Every thing is going well on git, except git push -u "...". Each time the following message appears " remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
The internet link does not work and I do not understand anything about that.
Please help !!!!

**Comment by airtower-luna** [comment]
That means whatever you're giving as the password isn't valid. Note that you *cannot* use your password to push/fetch/pull, as described in that article.

When Git asks for the password you need to provide a [Personal Access Token (PAT)](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token). When creating the token make sure it has the **repo** scope, and additionally **workflow** if you use Actions.
